### ISSUE TITLE
Storage Robot Open Inventory with effect that "lowers robot"

### ISSUE THREAD

**Issue by Andrioden** [Andrioden/Robocodo#275]

**Comment by bjorntore** [comment]
OpenInventory instruction doesnt seem to do anything now? I think I need a demo of the storage robot in real life before I start doing this task. @Andrioden

**Comment by Andrioden** [comment]
Yes, probably you have to check what the lastExecutedInstruction and se if its open inventory, then animate. If the storage robot is in "lowered mode", then animation is not run.

Do it in Animation method, check Harvest animation method, use ShouldAnimationBePlayed()
